man/less-variables.xml

```diff
@@ -94,10 +94,12 @@
     <varlistentry id='colors'>
       <term><varname>$SYSTEMD_COLORS</varname></term>
 
-      <listitem><para>The value must be a boolean. Controls whether colorized output should be
-      generated. This can be specified to override the decision that <command>systemd</command> makes based
-      on <varname>$TERM</varname> and what the console is connected to.</para>
-      </listitem>
+      <listitem><para>Takes a boolear argument. When true, <command>systemd</command> and related utilities
+      will use colors in their output, otherwise the output will be b&w. Additionally, the variable can take
```

Comment:
hmm, `&` is special in xml, would need to be escaped.

but b&w is misleading, because it's not necessarily black and white... better say "monochrome" or so

Reply:
Fixed, thanks.